jetty-ee10/jetty-ee10-annotations/src/main/java/org/eclipse/jetty/ee10/annotations/AnnotationConfiguration.java

```diff
@@ -77,28 +74,36 @@ public class AnnotationConfiguration extends AbstractConfiguration
     public static final String CONTAINER_INITIALIZER_STARTER = "org.eclipse.jetty.containerInitializerStarter";
     public static final String MULTI_THREADED = "org.eclipse.jetty.annotations.multiThreaded";
     public static final String MAX_SCAN_WAIT = "org.eclipse.jetty.annotations.maxWait";
+    protected static final String STATE = "org.eclipse.jetty.annotations.state";
 
     public static final int DEFAULT_MAX_SCAN_WAIT = 60; /* time in sec */
     public static final boolean DEFAULT_MULTI_THREADED = true;
 
-    protected final List<AbstractDiscoverableAnnotationHandler> _discoverableAnnotationHandlers = new ArrayList<>();
-    protected ClassInheritanceHandler _classInheritanceHandler;
-    protected final List<ContainerInitializerAnnotationHandler> _containerInitializerAnnotationHandlers = new ArrayList<>();
-    protected final List<DiscoveredServletContainerInitializerHolder> _sciHolders = new ArrayList<>();
-
-    protected List<ParserTask> _parserTasks;
+    protected static class State
```

Comment:
Is this a `record` ?

Reply:
I was intended to do that, but I'd need to make all fields final, which is a bit of work that risks changing behaviour.